app/views/users/logix/edit.html.erb

```diff
@@ -1,46 +1,57 @@
-
-
+<link href="/css/typeahead.css" rel="stylesheet">
```

Comment:
Can we use minimised version if we are using css directly

Reply:
i have added the minified css.